web/src/compose_recipient.ts

```diff
@@ -439,7 +439,10 @@ export function update_topic_displayed_text(topic_name = "", has_topic_focus = f
     }
 
     const is_empty_string_topic = compose_state.topic() === "";
-    if (is_empty_string_topic && !has_topic_focus && recipient_widget_hidden) {
+    if (
+        $input.prop("disabled") ||
```

Comment:
Why is this the right check?

Is the `stream_data.is_empty_topic_only_channel(compose_state.stream_id())` if statement above the only situations where this input gets disabled? If so, I guess I'm not sure why we'd check this variable rather than that more data-focused check -- here we're implicitly assuming the only reason that input would be disabled is if it's a general-chat channel.